appsec-kit-demo-v8-spring/pom.xml

```diff
@@ -128,6 +133,11 @@
 
 	<build>
 		<plugins>
+				<plugin>
+						<groupId>org.springframework.boot</groupId>
+						<artifactId>spring-boot-maven-plugin</artifactId>
+						<version>2.2.0.RELEASE</version>
```

Comment:
Should we use the `spring.boot2.version` here too?

Reply:
sone